core/css.js

```diff
@@ -14,28 +14,29 @@
  * @name Blockly.Css
  * @namespace
  */
-goog.provide('Blockly.Css');
+goog.module('Blockly.Css');
+goog.module.declareLegacyNamespace();
 
 
 /**
  * Has CSS already been injected?
  * @type {boolean}
  * @private
  */
-Blockly.Css.injected_ = false;
+let injected_ = false;
```

Comment:
I was under the impression that we also wanted to remove underscores (and possibly the @private? A bit unclear if we've decided on that yet)

Reply:
Fixed.